src/Resources/contao/widgets/FileTree.php

```diff
@@ -99,7 +99,7 @@ protected function validator($varInput)
 		// Store the order value
 		if ($this->orderField != '')
 		{
-			$arrNew = array_map('StringUtil::uuidToBin', explode(',', \Input::post($this->strOrderName)));
+			$arrNew = !empty($order = \Input::post($this->strOrderName)) ? array_map('StringUtil::uuidToBin', explode(',', $order)) : array();
```

Comment:
I don't like the coding style here in terms of readability. What about this?

```php
$arrNew = array();

if ($order = \Input::post($this->strOrderName))
{
    $arrNew = array_map('StringUtil::uuidToBin', explode(',', $order));
}
```

Reply:
Yeah, that's easier to read. I will change this tomorrow.

Reply:
Never mind, I already have so I can merge the PR.